src/components/spot/swiper/index.tsx

```diff
@@ -27,64 +18,73 @@ const Swiper = () => {
 
   const moveToNthSlide = (index: number) => {
     setTimeout(() => {
-      setItemNumber(index);
+      setSlideNumber(index);
       setIsEndSlide(true);
     }, 500);
   };
 
   const clickRightArrow = () => {
     if (slideNumber === carouselList.length - 2) {
-      setItemNumber(slideNumber + 1);
+      setSlideNumber(slideNumber + 1);
       moveToNthSlide(1);
     } else {
       if (isEndSlide) setIsEndSlide(false);
-      setItemNumber(slideNumber + 1);
+      setSlideNumber(slideNumber + 1);
     }
   };
 
   const clickLeftArrow = () => {
     if (slideNumber === 1) {
-      setItemNumber(slideNumber - 1);
+      setSlideNumber(slideNumber - 1);
       moveToNthSlide(carouselList.length - 2);
     } else {
       if (isEndSlide) setIsEndSlide(false);
-      setItemNumber(slideNumber - 1);
+      setSlideNumber(slideNumber - 1);
     }
   };
 
   return (
     <SwiperWrapper>
       <LeftArrow onClick={() => clickLeftArrow()}>&lt;</LeftArrow>
       <Slide slideNumber={slideNumber} endSlide={isEndSlide}>
-        {carouselList.map((store) => (
-          <SlideItem
-            key={store.key}
-            category={store.category}
-            title={store.title}
-            address={store.address}
-          />
-        ))}
+        {carouselList.map((store, index) => {
+          let isCenter = false;
+          if (index === slideNumber) isCenter = true;
+          return (
+            <SlideItem
+              key={store.key}
+              category={store.category}
+              title={store.storeName}
+              address={store.pickUpLocation}
+              center={isCenter}
+              endSlide={isEndSlide}
+            />
+          );
```

Comment:
아래처럼 작성해 볼 수도 있을거 같네요

가급적이면 `let`을 사용하지 않고 작성해보시는걸 추천 드리고 싶습니다 ㅎㅎ

```suggestion
          return (
            <SlideItem
              key={store.key}
              category={store.category}
              title={store.storeName}
              address={store.pickUpLocation}
              center={index === slideNumber}
              endSlide={isEndSlide}
            />
          );
```

Reply:
아하 넵 ! 간단히 처리 되네용...